src/documentation/pages/components/forms/range-slider/examples/_multiple_labels.vue

```diff
@@ -0,0 +1,62 @@
+<template lang="pug">
+  <ComponentExample title="Multiple Labels" id="multiple-labels" :tabs="exampleTabs">
+    .chi-form__item(slot="example")
+      label(class="chi-label", id="range-label", for="range") Range label
+      .chi-input__wrapper.-d--flex
+        span.-text.-mr--2 200GB
+        input(type="range", class="chi-range-slider", ref="range")
```

Comment:
On label you are using `for="range03"`
```suggestion
        input(type="range", class="chi-range-slider", ref="range" id="range03")
```